app/index.js

```diff
@@ -653,6 +657,33 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    promptConfigurePushNotifications: function() {
+      if (this.skipPrompting) return;
+      if (!this.servicesToConfigure) return;
+      if (!this.servicesToConfigure.pushnotifications) return;
+      var done = this.async();
+
+      this.log();
+      this.log('Configure Push Notifications');
+      var prompts = [
+        { name: 'pushNotificationsName', message: 'Enter service name (blank for default):',
+          when: (answers) => this.bluemix
+        },
+        { name: 'pushNotificationsRegion', message: 'Enter Bluemix region (blank for default of us_south):' },
```

Comment:
Might even be best to have a list for the user to choose from rather than them typing it in.